plugin.go

```diff
@@ -123,11 +124,15 @@ func getModules() *exec.Cmd {
 	)
 }
 
-func planCommand(variables map[string]string, secrets map[string]string, parallelism int) *exec.Cmd {
+func planCommand(variables map[string]string, secrets map[string]string, parallelism int, targets []string) *exec.Cmd {
 	args := []string{
 		"plan",
 		"-out=plan.tfout",
 	}
+	for _, v := range targets {
+		args = append(args, "--target")
```

Comment:
This line can be combined with the next to be `args = append(args, "--target", fmt.Sprintf("%s", v)) `